bloom-housing/bloom
# 1780/Cleanup getStaticProps and getStaticPaths

## Issue

Addresses #1780 


## Description

The cache revalidation piece doesn't actually change here, the main changes are is so that errors aren't swallowed on builds and fails them (should cover #1618) and I removed the export script, since it isn't valid with [fallback: true](https://nextjs.org/docs/advanced-features/static-html-export#caveats). So we'll have to change the build command to replace `export` with `start`.

This doesn't address the second part of #1780, which is to add applicationOpenDate to the form and check against that before displaying, since that feels separate enough to warrant another PR.

## Type of change


## How Can This Be Tested/Reviewed?

I tested locally by starting the back end, then building and starting public. If you have partners started, you should be able to add/edit a listing and after `CACHE_REVALIDATE` (set in your .env) expires, you should see the update after refreshing. You should also be able to stop the backend and continue to see listings and go to detail pages.


## Checklist:
